content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -176,7 +176,7 @@ If you are on Linux and using [Homebrew](https://docs.brew.sh/Homebrew-on-Linux)
 
 ```shell
 brew install kubectl
-kubectl version --client
+kubectl version --short
```

Comment:
Why is this better? (It produces a warning now and will produce an error in the future).